packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/fragment.js

```diff
@@ -150,6 +150,10 @@ function is_static_element(node) {
 			return false;
 		}
 
+		if (node.name === 'img' && attribute.name === 'loading') {
```

Comment:
Could you add a comment here explaining what the issue is?

Reply:
Yup

Reply:
Done, btw @benmccann this is the same function we need to change to fix #14241 so if you are around and want to merge this it will be less conflict prone 😄

Reply:
I'm afraid I'm still missing something. Why isn't an img with loading lazy static?

Reply:
Because the produced code needs to get a reference to the image element so that it can be passed to handle_lazy_image